index.html

```diff
@@ -0,0 +1,726 @@
+                        <h4 class="article-title">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ab perferendis culpa alias error dicta!</h4>
+                    </div>
+                    <div class="clearfix border-bottom pb-3 pt-3">
+                        <img class="float-left" src="./images/section-highlights-D04.jpg" alt="image">
+                        <h4 class="article-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Alias, molestias?</h4>
+                    </div>
+                    <div class="clearfix pb-3 pt-3">
+                        <img class="float-left" src="./images/section-highlights-D05.jpg" alt="image">
+                        <h4 class="article-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi!</h4>
+                    </div>
+                </div>
+            </div>
+            <div class="row magazine-section">
+                <h3 class="main-title text-danger col-xs-12 col-sm-12 border-bottom mt-2">SECTION E</h3>
+                <div class="card col-xs-12 col-sm-12 col-md-4 border-0">
+                    <img src="./images/section-highlights-E01.jpg" alt="image" class="card-img-top">
+                    <div class="card-body pl-0 pr-0">
+                        <h2 class="main-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ad, commodi omnis.</h2>
+                    </div>
+                </div>
+                <div class="card col-xs-12 col-sm-12 col-md-4 border-0">
+                    <img src="./images/section-highlights-E02.jpg" alt="image" class="card-img-top">
+                    <div class="card-body pl-0 pr-0">
+                        <h2 class="main-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus, perspiciatis?</h2>
+                    </div>
+                </div>
+                <div class="other-section-stories col-xs-12 col-sm-12 col-md-4 border-0">
+                    <div class="clearfix border-bottom pb-3">
+                        <img class="float-left" src="./images/section-highlights-E03.jpg" alt="image">
+                        <h4 class="article-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium, ad. Blanditiis.</h4>
+                    </div>
+                    <div class="clearfix border-bottom pb-3 pt-3">
+                        <img class="float-left" src="./images/section-highlights-E04.jpg" alt="image">
+                        <h4 class="article-title">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi?</h4>
+                    </div>
+                    <div class="clearfix pb-3 pt-3">
+                        <img class="float-left" src="./images/section-highlights-E05.jpg" alt="image">
+                        <h4 class="article-title">Lorem ipsum dolor sit amet consectetur adipisicing.</h4>
+                    </div>
+                </div>
+            </div>
+            <div class="row magazine-section">
```

Comment:
Let's take this last part before the row for example.
```
<section>
    <h3>SECTION F</h3>
    <div class="d-flex">
       .....
    </div>
</section>
```

Reply:
d-flex is not needed. I have achieved the same effect using grid. The appearance looks fine to me. Even if I added a container that uses `d-flex` I will still have to give a width to each article using grid sizing (`col-md-4` etc). 

Explain, why you want this to be changed first. That is how is the behaviour of my code different from desired, and how would this change help.